cmd/backup.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/tigrisdata/tigris-cli/client"
+	"github.com/tigrisdata/tigris-cli/util"
+	"github.com/tigrisdata/tigris-client-go/driver"
+)
+
+func listDatabases(ctx context.Context, dbFilter []string) ([]string, error) {
+	databases := []string{}
+
+	resp, err := client.Get().ListDatabases(ctx)
+	if err != nil {
+		return databases, util.Error(err, "list databases")
```

Comment:
if you return error then you should return nil instead of `databases`.
e.g: `return nil, err`

Reply:
Sounds good! So instead of an empty []string{} just return nil, right?